feature-libs/cart/wish-list/core/facade/wish-list.service.ts

```diff
@@ -124,4 +98,19 @@ export class WishListService implements WishListFacade {
   protected getWishListId(): Observable<string> {
     return this.multiCartFacade.getCartIdByType(CartType.WISH_LIST);
   }
+
+  private getWishListIdWithUserId(): Observable<string[]> {
```

Comment:
Could it be protected?

Reply:
This new function just extract the duplicate code from `addEntry` and `removeEntry`. It should only be used in `addEntry` and `removeEntry`.